### PR TITLE
[bitnami/redis-cluster] Add priorityClassName to initJob and updateJob

### PR DESCRIPTION
**Pull request by mboutet**
**Description of the change**

Add `priorityClassName` to `initJob` and `updateJob`.

**Benefits**

Prevent the jobs to be evicted by higher priority pods and thus causing Helm to fail because it thinks the hook has failed.

**Possible drawbacks**

None.

**Applicable issues**

None.

**Additional information**

None.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/redis-cluster/values-production.yaml [hunk 1]
```diff
@@ -365,6 +370,11 @@ updateJob:
   ## Redis pod/node affinity/anti-affinity
   ##
   affinity: {}
+
+  ## Priority class name
+  ##
+  # priorityClassName: ""
```
Comment (andresbono) [inline bitnami/redis-cluster/values-production.yaml:376]:
```suggestion
  priorityClassName:
```

File: bitnami/redis-cluster/values-production.yaml [hunk 2]
```diff
@@ -280,6 +280,11 @@ initJob:
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   ##
   affinity: {}
+
+  ## Priority class name
+  ##
+  # priorityClassName: ""
```
Comment (andresbono) [inline bitnami/redis-cluster/values-production.yaml:286]: Does it make sense to uncomment the parameter and leave it as nil? The same than `redis.priorityClassName`:

```suggestion
  priorityClassName:
```

File: bitnami/redis-cluster/values.yaml [hunk 3]
```diff
@@ -280,6 +280,11 @@ initJob:
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   ##
   affinity: {}
+
+  ## Priority class name
+  ##
+  # priorityClassName: ""
```
Comment (andresbono) [inline bitnami/redis-cluster/values.yaml:286]:
```suggestion
  priorityClassName:
```

File: bitnami/redis-cluster/values.yaml [hunk 4]
```diff
@@ -365,6 +370,11 @@ updateJob:
   ## Redis pod/node affinity/anti-affinity
   ##
   affinity: {}
+
+  ## Priority class name
+  ##
+  # priorityClassName: ""
```
Comment (andresbono) [inline bitnami/redis-cluster/values.yaml:376]:
```suggestion
  priorityClassName:
```

**Comment by andresbono** [comment]
Thanks so much for providing this PR, @mboutet!! Could you please check my suggestions?

**Comment by mboutet** [comment]
I applied your suggestions

**Review by andresbono — APPROVED** [review]
LGTM. Thanks for your contribution!!

**Pull request merged**